### ISSUE TITLE
License

### ISSUE THREAD

**Issue by berezovskyi** [labra/umlShaclex#1]
https://opensource.stackexchange.com/questions/1640/if-im-using-a-gpl-3-library-in-my-project-can-i-license-my-project-under-mit-l says that anything that uses GPL becomes GPL. Do you think that MIT is appropriate for `umlShaclex`?

**Comment by labra** [comment]
You may be right.

However, when I was going to change the umlShaclex license to GPL, I read the [plantuml FAQ](http://plantuml.com/faq) and they have this answer:  

> I don't like GPL! 
> You can also use:
> LGPL license
> Apache license
> Eclipse Public license
> MIT license
> Those versions miss few features (DITAA for example), but are 100% able to generate UML diagrams.

So it seems that it is possible to use a non-GPL version of plantuml. 

I need to see if I can choose the non-GPL version from Maven (maybe I should include the jar directly).

**Comment by berezovskyi** [comment]
Yes, I checked this on maven too but there seems to be no non-GPL version on Maven. See http://forum.plantuml.net/7105/apache-or-mit-in-mvn-repository?show=7105#q7105
